specification/orbital/resource-manager/Microsoft.Orbital/aodg/preview/2024-03-01-preview/aodg.json

```diff
@@ -0,0 +1,1750 @@
+      "get": {
+        "tags": [
+          "Edge Site"
+        ],
+        "description": "Returns a list of edge sites for a resource group.",
+        "operationId": "EdgeSites_List",
+        "x-ms-examples": {
+          "List of Edge Sites by Resource Group": {
+            "$ref": "./examples/EdgeSitesByResourceGroupList.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/skipTokenParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of edge site resources.",
+            "schema": {
+              "$ref": "#/definitions/EdgeSiteListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the resource group does not exist, 404 (NotFound) will be returned.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Orbital/l2Connections/{l2ConnectionName}": {
+      "get": {
+        "tags": [
+          "L2 Connection"
+        ],
+        "operationId": "L2Connections_Get",
+        "description": "Gets the specified L2 connection in a specified resource group.",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/l2ConnectionNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns the resulting L2Connection resource.",
+            "schema": {
+              "$ref": "#/definitions/L2Connection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
```

Comment:
>"$ref": "#/definitions/CloudError" [](http://example.com/codeflow?start=14&length=34)

Use "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse" at all places wherever you have used CloudError definition and remove the definition from this swagger file.